src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/RuntimeFeature.cs

```diff
@@ -18,6 +18,14 @@ public static class RuntimeFeature
         public const string DefaultImplementationsOfInterfaces = nameof(DefaultImplementationsOfInterfaces);
 #endif
 
+#if CORERT
+        public static bool IsDynamicCodeSupported => false;
```

Comment:
I assume that once/if we have more complete support for JIT or IL interpreter in CoreRT, we will figure a scheme to override these values.

Reply:
Would Mono want to share this file too? Maybe we could just make RuntimeFeature a partial class and put the two properties in the platform-specific parts.

The references to the properties from the `IsSupported` API will make sure nobody forgets to add a dot file that defines them because it won't compile otherwise.

Reply:
Make sense